enterprise-ws-spec/src/main/asciidoc/ClientProgrammingModel.adoc

```diff
@@ -137,39 +131,38 @@ Service abf = (Service)ic.lookup(
 "java:comp/env/service/AddressBookService");
 
 In the above example, the container must ensure that an implementation
-of the generic Service Interface, javax.xml.rpc.Service or generic
-Service class, javax.xml.ws.Service, is bound in the JNDI name space at
-a location specified by the developer. A similar code fragment is used
-for access to an object that implements a Generated Service Interface
-such as AddressBookService.
+of the generic Service Interface, jakarta.xml.ws.Service, is bound in the
```

Comment:
`jakarta.xml.ws.Service` is a _generic Service class_, not an interface. This must be fixed.